wallet-new/server/Cardano/Wallet/Server/CLI.hs

```diff
@@ -1,22 +1,23 @@
+{-# LANGUAGE ApplicativeDo #-}
```

Comment:
Sorry to be a pain in the ass, but any chance we can *not* use `ApplicativeDo`? It feels a little weird that we are using a package like `optparse-applicative` abusing the `do` notation. I know we are doing this virtually everywhere in cardano, but I thought that for new code we could try something different, if possible 😉

Reply:
When I've read this ^, I realized what was wrong :))
Arguments were passed in wrong order to `TlsParams` constructor